docs/docs/reference/metaprogramming/tasty-reflect.md

```diff
@@ -117,17 +116,17 @@ def macroImpl()(qctx: QuoteContext): Expr[Unit] = {
 ### Tree Utilities
 
 `scala.tasty.reflect` contains three facilities for tree traversal and
-transformations.
+transformation.
 
 `TreeAccumulator` ties the knot of a traversal. By calling `foldOver(x, tree))`
-we can dive in the `tree` node and start accumulating values of type `X` (e.g.,
+we can dive into the `tree` node and start accumulating values of type `X` (e.g.,
 of type List[Symbol] if we want to collect symbols). The code below, for
 example, collects the pattern variables of a tree.
 
 ```scala
-def collectPatternVariables(tree: Tree)(implicit ctx: Context): List[Symbol] = {
+def collectPatternVariables(tree: Tree)(implicit qctx: QuoteContext): List[Symbol] = {
```

Comment:
please check this one carefully, it was more a guess that it should be QuoteContext

Reply:
Actually it is `Context` (the equivalent to a `qctx.tasty.Context`)

Reply:
reverted.